snapcraft/internal/pluginhandler.py

```diff
@@ -644,7 +651,7 @@ def _migratable_filesets(fileset, srcdir):
 
 
 def _migrate_files(snap_files, snap_dirs, srcdir, dstdir, missing_ok=False,
-                   follow_symlinks=False):
+                   follow_symlinks=False, fixup_func=None):
```

Comment:
Don't you want to make it a lambda function taking one parameter? That way, it's clear from the signature that the function is of form:
```
def fun(dst):
  pass
```
As you don't have docstrings to specify it.